lightning/src/routing/network_graph.rs

```diff
@@ -674,6 +674,21 @@ impl ChannelInfo {
 		};
 		Some((DirectedChannelInfo { channel: self, direction }, source))
 	}
+
+	/// Returns a [`DirectedChannelInfo`] for the channel coming from the given `source` to a
+	/// returned `target`, or `None` if `source` is not one of the channel's counterparties.
+	pub fn as_coming_from(&self, source: &NodeId) -> Option<(DirectedChannelInfo, &NodeId)> {
```

Comment:
Would prefer to name this `as_directed_from` for consistency.

Reply:
Sure, addressed with https://github.com/lightningdevkit/rust-lightning/pull/1286/commits/d8fe725ae98226dad4aa3c37d7d5650ccdd13ad9.